pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -0,0 +1,73 @@
+//go:build policycoreutils
+// +build policycoreutils
+
+package policycoreutils
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/containers/selinuxd/pkg/semodule/seiface"
+	"github.com/go-logr/logr"
+)
+
+type SEModulePcuHandler struct {
+	logger logr.Logger
+}
+
+// Ensure that the test handler implements the Handler interface
+var _ seiface.Handler = &SEModulePcuHandler{}
+
+func runSeModule(opFlag, policyArg string) (string, error) {
+	cmd := exec.Command("/usr/sbin/semodule", opFlag, policyArg)
```

Comment:
Could you use `CommandContext` instead? This way we can pass a `context.Context` object and potentially do timeouts. It could be `context.TODO` to begin with and we can leave for later (a low hanging fruit bug would be ideal for newcomers to the project).

Reply:
Also... should you make this call verbose so we can get more info on potential issues?